gapic/schema/wrappers.py

```diff
@@ -821,13 +843,22 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    @property
+    def http_options(self) -> List[HttpRule]:
+        """Return a list of the http bindings for this method."""
+        http = self.options.Extensions[annotations_pb2.http]
+        http_options = [http] + list(http.additional_bindings)
+        opt_gen = (HttpRule.try_parse_http_rule(http_rule)
+                   for http_rule in http_options)
+        return [rule for rule in opt_gen if rule]
+
     @property
     def http_opt(self) -> Optional[Dict[str, str]]:
```

Comment:
I agree that this property should be removed - I would prefer to do this in a follow-up PR, after fixing the rest template that currently relies on it.

Reply:
Fine by me